groth16.nimble

```diff
@@ -7,5 +7,5 @@ skipDirs    = @["groth16/example"]
 binDir      = "build"
 namedBin    = {"cli/cli_main": "nim-groth16"}.toTable()
 
-requires "https://github.com/status-im/nim-taskpools"
+requires "https://github.com/status-im/nim-taskpools >= 0.0.5"
```

Comment:
Taskpool 0.0.4 has some odd compilation issues, 0.0.5 seems to have fixed something. So I'd recommend it.